docs/tutorial.rst

```diff
@@ -1025,16 +1018,86 @@ We'll count up the number of neutral and deleterious mutations in the three regi
             f"and {d} deleterious mutations."
         )
 
-    # From 0 to 1000: 3 neutral mutations and 0 deleterious mutations.
-    # From 1000 to 2000: 1 neutral mutations and 2 deleterious mutations.
-    # From 2000 to 3000: 1 neutral mutations and 0 deleterious mutations.
+    # From 0 to 1000: 50 neutral mutations and 0 deleterious mutations.
+    # From 1000 to 2000: 12 neutral mutations and 20 deleterious mutations.
+    # From 2000 to 3000: 45 neutral mutations and 0 deleterious mutations.
 
 This verifies that there are only deleterious mutations in the center bit.
 
 
+.. _sec_tute_selection_annotation:
+
+3. Simulating selection on exons
+--------------------------------
 
+The catalog also has a certain number of *annotations* available,
+obtained from Ensembl.
+For instance, for humans we have:
+
+.. code-block:: python
 
-3.  Selective sweep
+    for a in species.annotations:
+        print(f"{a.id}: {a.description}")
+
+    # ensembl_havana_104_exons: Ensembl Havana exon annotations on GRCh38
+    # ensembl_havana_104_CDS: Ensembl Havana CDS annotations on GRCh38
+
+To simulate with the HomSap/Gamma_K17 DFE, applied this time
+to *all* exons on chromosome 21
+(the remainder of the chromosome will have only neutral mutations),
+we extract the intervals from the :class:`.Annotation` object
+and use this in :meth:`.Contig.add_DFE`:
+
+.. code-block:: python
+
+    species = stdpopsim.get_species("HomSap")
+    dfe = species.get_dfe("Gamma_K17")
+    contig = species.get_contig("chr20")
+    model = species.get_demographic_model("OutOfAfrica_3G09")
+    samples = model.get_samples(100, 100, 100)  # YRI, CEU, CHB
+
+    exons = species.get_annotations("ensembl_havana_104_exons")
+    exon_intervals = exons.get_chromosome_annotations("chr20").astype("int")
+    contig.add_DFE(intervals=exon_intervals, DFE=dfe)
+
+    engine = stdpopsim.get_engine("slim")
+    ts = engine.simulate(
+        model,
+        contig,
+        samples,
+        seed=236,
+        slim_scaling_factor=100,
+        slim_burn_in=10,
+    )
+
+Note the very large scaling factor ($Q=100$) that we've used here to get this
+to run fast enough to be used for a quick example!
+This is *not* expected to be a good example because of this extreme scaling,
+but nonetheless there is lower diversity in exons than outside of them:
+
+.. code-block:: python
+
+    breaks, labels = contig.dfe_breakpoints()
```

Comment:
Note how the `dfe_breakpoints()` method gives us just what we need to pass to the tree sequence statistics.

Reply:
aha! nice!